.github/workflows/ci.yml

```diff
@@ -63,3 +66,100 @@ jobs:
     - name: Build kube-router
       run: |
         make kube-router
+
+  # Builds Container on Push Event, not run if there's a Tag Event
+  ci-build-container-push:
+    runs-on: ubuntu-latest
+    if: ${{ startsWith('refs/tags/v', github.ref) != true && github.event_name != 'pull_request'  }}
+    steps:
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        platforms: linux/amd64,linux/arm64,linux/arm,linux/ppc64,linux/s390x
+        push: true
+        tags: cloudnativelabs/kube-router:${{ github.sha }}
```

Comment:
Might it be better to use the `github.ref` here as well? Refs are easier to find on dockerhub than the sha.

Also, I believe that these go into the `kube-router-git` registiry right?

Reply:
Yes, you are right. Changed it. Thanks!